app/scripts/background.js

```diff
@@ -65,11 +69,34 @@ async function loadStateFromPersistence () {
   // migrations
   const migrator = new Migrator({ migrations })
   // read from disk
-  let versionedData = diskStore.getState() || migrator.generateInitialState(firstTimeState)
+  versionedData = diskStore.getState() || migrator.generateInitialState(firstTimeState)
```

Comment:
maybe something like
`versionedData = localData || diskStore.getState() || migrator.generateInitialState(firstTimeState)`

Reply:
New commit moves storage.local loading before this assignment so I can use a similar format to what you suggest.